website/src/pages/tutorial.md

```diff
@@ -0,0 +1,530 @@
+
+```rust {1-2}
+- #[derive(Clone, PartialEq)]
++ #[derive(Clone, PartialEq, Deserialize)]
+struct Video {
+    id: usize,
+    title: String,
+    speaker: String,
+    url: String,
+}
+```
+
+Now as the last step, we need to update our `App` component to make the fetch request instead of using hardcoded data
+
+```rust {3-23,32-33}
+#[function_component(App)]
+fn app() -> Html {
+-    let videos = vec![
+-        // ...
+-    ]
++    let videos = use_state(|| vec![]);
++    {
++        let videos = videos.clone();
++        use_effect(move || {
++            let videos = videos.clone();
++            wasm_bindgen_futures::spawn_local(async move {
++                let fetched_videos: Vec<Video> = Request::get("https://yew.rs/tutorial/data.json")
++                    .send()
++                    .await
++                    .unwrap()
++                    .json()
++                    .await
++                    .unwrap();
++                videos.set(fetched_videos);
++            });
++            || ()
++        });
++    }
+
+    // ...
+ 
+    html! {
+        <>
+            <h1>{ "RustConf Explorer" }</h1>
+            <div>
+                <h3>{"Videos to watch"}</h3>
+-                <VideosList videos={videos} on_click={on_video_select.clone()} />
++                <VideosList videos={(*videos).clone()} on_click={on_video_select.clone()} />
+            </div>
+            { for details }
+        </>
+    }
+}
+```
+
+:::note
+We're using `unwrap`s here because it is a demo application. In a real world, you want to have proper error handling.
+:::
+
+Now look at the browser to see everything working as expected... Which would've been the case if it weren't for CORS.
+In order to fix that, we need a proxy server. Luckily trunk provides that.
+
+Update the following line:
+
+```rust {2-3}
+// ...
+-                let fetched_videos: Vec<Video> = Request::get("https://yew.rs/tutorial/data.json")
++                let fetched_videos: Vec<Video> = Request::get("/tutorial/data.json")
+// ...
+```
+
+Now, rerun the server with the following command:
+
+```bash
+trunk serve --proxy-backend=https://yew.rs/tutorial
+```
+
+Refresh the tab and incredible, everything works as expected.
+
+## Wrapping up
+
+Congratulations! You’ve created a web application that fetches data from external API and displays a list of videos.
+
+## What's next
+
+Obviously, this application is very far from perfect or useful. After going through this tutorial, 
+you can use it as a jumping-off point to explore more advanced topics.
+
+### Styles
+
+Our apps look very ugly. There's no CSS, or any kind of styles.
+Unfortunately, Yew doesn't offer a built-in way to style components. See [Trunk's assets](https://trunkrs.dev/assets/)
+to learn how to add style sheets.
+
+### More libraries
+
+Our app made use of only a few external dependencies. There are lots of crates out there that can be used.
+See [external libraries](/next/more/external-libs) for more details.
```

Comment:
Should `/next/` be hardcoded here?

Reply:
For now, it has to be. We can fix that after the next release. Links default to the latest (0.18 in this case) version and that page doesn't exist there.